### PR TITLE
[DUOS-3012] Add Terra data library

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-3012

### Summary

Add Terra data library

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/images/terra-logo.svg
Comment (aarohinadkarni) [inline src/images/terra-logo.svg:1]: nit: logo seems a bit smaller compared to other branded data libraries we have

**Review by aarohinadkarni — APPROVED** [review]
👍

**Review by PintoGideon — APPROVED** [review]
Tested and looks good.

File: src/images/terra-logo.svg
Comment (fboulnois) [reply src/images/terra-logo.svg:1]: Yes. I don't have control over this logo unfortunately, it comes directly from Terra.

**Review by rushtong — APPROVED** [review]
👍🏽

**Pull request merged**